Documentation/Troubleshooting/disaster-recovery.md

```diff
@@ -260,73 +259,120 @@ the CRs to their prior state without even necessarily suffering cluster downtime
 2.  Backup all Rook CRs and critical metadata
 
     ```console
-    # Store the CephCluster CR settings. Also, save other Rook CRs that are in terminating state.
+    # Store the `CephCluster` CR settings. Also, save other Rook CRs that are in terminating state.
     kubectl -n rook-ceph get cephcluster rook-ceph -o yaml > cluster.yaml
 
     # Backup critical secrets and configmaps in case something goes wrong later in the procedure
     kubectl -n rook-ceph get secret -o yaml > secrets.yaml
     kubectl -n rook-ceph get configmap -o yaml > configmaps.yaml
     ```
 
-3.  Remove the owner references from all critical Rook resources that were referencing the CephCluster CR.
-    The critical resources include:
+3.  Back up and remove the `ValidatingWebhookConfiguration`. This is the resource which connects Rook custom resources to the
+    operator pod's validating webhook. Because the operator is unavailable, we must temporarily disable the valdiating webhook in
+    order to make changes.
 
-      - Secrets: `rook-ceph-admin-keyring`, `rook-ceph-config`, `rook-ceph-mon`, `rook-ceph-mons-keyring`
-      - ConfigMap: `rook-ceph-mon-endpoints`
-      - Services: `rook-ceph-mon-*`, `rook-ceph-mgr-*`
-      - Deployments: `rook-ceph-mon-*`, `rook-ceph-osd-*`, `rook-ceph-mgr-*`
-      - PVCs (if applicable): `rook-ceph-mon-*` and the OSD PVCs (named `<deviceset>-*`, for example `set1-data-*`)
+    1.  Save the `ValidatingWebhookConfiguration` responsible for controlling Rook resources. (These are not namespaced
+        resources.)
 
-    For example, remove this entire block from each resource:
+        ```console
+        kubectl get ValidatingWebhookConfiguration rook-ceph-webhook -o yaml > rook-ceph-webhook.yaml
+        ```
 
-    ```yaml
-    ownerReferences:
-    - apiVersion: ceph.rook.io/v1
-       blockOwnerDeletion: true
-       controller: true
-       kind: CephCluster
-       name: rook-ceph
-       uid: <uid>
-    ```
+    2.  Delete the `ValidatingWebhookConfiguration`.
```

Comment:
while creating webhook, we also create `certificate` and `Issuers`
do we also need to remove those? right?

https://github.com/rook/rook/blob/7f0c83ad18346e8880a168468899be0ec55bc18a/pkg/operator/ceph/webhook-config.go#L115-L133

https://github.com/rook/rook/blob/7f0c83ad18346e8880a168468899be0ec55bc18a/pkg/operator/ceph/webhook-config.go#L61-L73

Reply:
I don't believe it's necessary to remove those (at least, I didn't). Removing the `ValidatingWebhookConfiguration` and then restoring it before scaling back up the operator pod is sufficient to keep Kubernetes from blocking CR edits.